packages/editor/src/di.config.ts

```diff
@@ -94,32 +87,24 @@ export default function createContainer(widgetId: string, ...containerConfigurat
     ivyConnectorModule,
     ivyKeyListenerModule,
     ivyNotificationModule,
-    ivyMarqueeSelectionToolModule,
-
-    // Ivy extensions:
-    ivyHelperLineExtensionModule,
-    ivyBoundsExtensionModule,
 
     // additional configurations
     ...containerConfiguration
   );
 
   // configurations
+  bindOrRebind(container, TYPES.Grid).toConstantValue({ x: 8, y: 8 });
   container.bind<IvyViewerOptions>(IVY_TYPES.IvyViewerOptions).toConstantValue(defaultIvyViewerOptions());
   container.bind<IHelperLineOptions>(TYPES.IHelperLineOptions).toConstantValue({
-    alignmentEpsilon: 0, // positions must match perfectly, we already restrict movement to the grid and only use integer positions (no decimals)
-    minimumMoveDelta: { x: IvyGridSnapper.GRID.x * 2, y: IvyGridSnapper.GRID.y * 2 },
-    alignmentElementFilter: element =>
-      !(element instanceof LaneNode) &&
-      !(element.id === feedbackEdgeId(element.root)) &&
-      !(element.id === feedbackEdgeEndId(element.root)) &&
```

Comment:
Yes, those two are now part of the default since they are only part of the feedback: https://github.com/eclipse-glsp/glsp-client/blob/65b68e37b8d18852c6a8a4a81f5db5f716ef926b/packages/client/src/features/helper-lines/helper-line-feedback.ts#L75